Sources/Parser/Parser.swift

```diff
@@ -91,6 +91,31 @@ public class Parser {
     return first
   }
 
+  // TEMP: UNTIL FOR LOOPS MERGED
```

Comment:
What is this referring to?

Reply:
Ah! Forgot about that. It's refering to that function being duplicated in this PR and in for loops